### DISCUSSION TITLE
Relative links generating warning but seem to be correct

### DISCUSSION THREAD

**Discussion by hmf** [planet42/Laika#234] [Q&A]
May be pushing it a bit here because I don't think it is a bug but I did not get any help from [Stackoverflow](https://stackoverflow.com/questions/57819028/laika-warning-unknown-link-for-relative-markdown-link-how-to-correct-it). So I will ask here: 

When running [Laika](https://planet42.github.io/Laika/using-laika/markup.html) on the following relative links (in Github flavor Markdown):

```
[CSS file](../css/docs.css) 
```
and
 
```
[MDoc](./mdoc.html)
```

I get these warnings:

```
warning: dev_notes/laika.md:9:1: Unknown link 'css/docs.css'.
[CSS file](../css/docs.css) of this documentation (look for the element `table`).
^^^^^^^^^^^^^^^^^^^^^^^^^^^
warning: dev_notes/laika.md:90:13: Unknown link 'dev_notes/mdoc.html'.
Here we use [MDoc](./mdoc.html)
            ^^^^^^^^^^^^^^^^^^^
```
The resulting HTML file seems to work correctly locally (browser loading files). How should I define the relative links to remove the warnings?

**Comment by hmf** [comment]
I have figured out that linking with an id so:

```
  [MDoc][./mdoc.md]
```
thee warning does not appear. However, this does not work linking to relative HTML and other file files.

**Comment by jenshalm** [comment]
For a direct link you need to use parentheses `( ... )` like in your first example. I don't think the warning you see comes from Laika, must be a different tool in your tool chain.

**Comment by hmf** [comment]
Oops. You are correct. Debug output from my Mill modules points to MDoc. Apologies for the noise. This is strange though, MDoc should only process code fences tagged with `mdoc`. Closing this issue.
